examples/pyfluent-workflow/main.py

```diff
@@ -155,34 +156,49 @@
 
 # # Set 150 iterations for solution
 
-fluent_session.solution.run_calculation.iter_count = 150
+fluent_session.solution.run_calculation.iterate(iter_count=150)
 
 # # Update Solution using WB Journal Command
 
 wb.run_script_string("system1 = GetSystem(Name=\"FLU\")")
 wb.run_script_string("solutionComponent1 = system1.GetComponent(Name=\"Solution\")")
 wb.run_script_string("solutionComponent1.Update(AllDependencies=True)")
 
-###############################################################################
-# # Create velocity vectors
-# # ~~~~~~~~~~~~~~~~~~~~~~~
-# # Create and display velocity vectors on the ``symmetry-xyplane`` plane
+# ## Post processing
+# Create and display velocity vectors on the ``symmetry-xyplane`` plane
 
-###############################################################################
-# # Post processing with PyVista (3D visualization)
-# # =============================================
+# ## Configure graphics picture export
+# Since Fluent is being run without the GUI, we will need to export plots as picture files. Edit the picture settings to use a custom resolution so that the images are large enough.
```

Comment:
```suggestion
# Since Fluent is being run without the GUI, you must export plots as picture files. Edit the picture settings to use a custom resolution so that the images are large enough.
```